Documentation/how-to-build-and-run-ilcompiler-in-console-shell-prompt.md

```diff
@@ -14,13 +14,11 @@ This will result in the following:
 - Build native and managed components of ILCompiler. The final binaries are placed to `<repo_root>\bin\<OS>.<arch>.<Config>\tools`.
 - Build and run tests
 
-# Install .NET Core 2.1 SDK
+## On Windows
 
-* Download .NET Core 2.1 SDK from [https://www.microsoft.com/net/download/core](https://www.microsoft.com/net/download/core)
-* On windows ensure you are using the 'x64 Native Tools Command Prompt for VS 2017'
-    (This is distinct from the 'Developer Command Prompt for VS 2017')
+- You should use `Developer Command Prompt for VS 2019`
```

Comment:
I do not think that the steps below work from `Developer Command Prompt for VS 2019`. I am getting the "module machine type 'X86' conflicts with target machine type 'x64'" error during the `dotnet publish` step. The problem is that `Developer Command Prompt for VS 2019` defaults to x86 tools, but the steps below assume x64. This should stay as `x64 Native Tools Command Prompt for VS 2019`.